core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -51,19 +50,6 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->append($this->addIntlNode())
                 ->variableNode('localconfig')
                     ->info('Allows to set TL_CONFIG variables, overriding settings stored in localconfig.php. Changes in the Contao back end will not have any effect.')
-                    ->validate()
-                        ->always(
-                            static function (array $options): array {
-                                foreach (array_keys($options) as $option) {
```

Comment:
Don't you think we'll deprecate more of them during 5.x?

Reply:
Restored in 7f5a64fa3ad7674bbc56c76911f7162433799619